### PR TITLE
SJQ2T-194 Endpoint for Create Bucket

### PR DESCRIPTION
**Pull request by MarkoRadunovicHtec**
Corresponding endpoint created, BucketCreateRequest class added, Validation for fields added, Swagger doc updated, isPublic field added to BucketResponse, Tests added

### CODE REVIEW

File: backend/src/main/java/com/htecgroup/skynest/model/response/BucketResponse.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  private UUID createdById;
+
+  private String name;
+
+  private UUID companyId;
+
+  private String description;
+
+  private Long size;
```
Comment (ivanfhtec) [inline backend/src/main/java/com/htecgroup/skynest/model/response/BucketResponse.java:18]: Should this response also contain isPublic?

**Review by DarkoVeljanoskiHtec — APPROVED** [review]
looks good

**Review by DraganMitrasinovicHtec — APPROVED** [review]
Legit

**Review by ivanfhtec — APPROVED** [review]
looks good

**Pull request merged**